pallets/parachain-staking/src/benchmarking.rs

```diff
@@ -145,13 +145,12 @@ benchmarks! {
 	on_initialize_network_rewards {
 		let issuance = T::Currency::total_issuance();
 		// if we only add by one, we also initialize a new year
-		let block = T::NetworkRewardStart::get() + T::BlockNumber::one() * 2_u64.into();
+		let block = T::NetworkRewardStart::get() + T::BlockNumber::one() * 2_u32.into();
 	}: { Pallet::<T>::on_initialize(block) }
 	verify {
 		let new_issuance = T::Currency::total_issuance();
-		let max_col_reward = InflationConfig::<T>::get().collator.reward_rate.per_block * MaxCollatorCandidateStake::<T>::get() * MaxSelectedCandidates::<T>::get().into();
+		let max_col_reward = InflationConfig::<T>::get().collator.reward_rate.per_block * MaxCollatorCandidateStake::<T>::get() * (MaxSelectedCandidates::<T>::get() as u64).into();
 		let network_block_reward = T::NetworkRewardRate::get() * max_col_reward;
-		assert!(new_issuance > issuance);
```

Comment:
This line was removed since our `NetworkRewardRate` is set to 0 (e.g. [here](https://github.com/pendulum-chain/pendulum/blob/991c7694084c22ee61716723d001163e746a1725/runtime/pendulum/src/lib.rs#L916))

Reply:
Instead of removing the check completely, maybe it would be better to use an if condition only 
```
if network_block_reward > 0 {
   assert!(new_issuance > issuance)
}
```